### PR TITLE
Complete calibrate app CI/CD pipeline

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
#### Summary of Changes (What does this PR do?)
- Adds the production pipeline
- Adds buildx to new website pipeline to speed up the build
- Corrects typos

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [x] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Please include the steps to be done inorder to setup and test this PR.

#### What are the relevant tickets?
- [PLAT-1070](https://airqoteam.atlassian.net/browse/PLAT-1070?atlOrigin=eyJpIjoiNzU2OGIyNmM0MWEwNDE2MmE2NTFiN2MwMGRhNWEwNjIiLCJwIjoiaiJ9)

#### Screenshots (optional)

### CODE REVIEW

File: .github/workflows/deploy-frontends-to-production.yml [hunk 1]
```diff
@@ -157,4 +167,56 @@ jobs:
...
+          kubeconfig: ${{ secrets.K8S_CONFIG_PROD }}
+
+      - name: NPM Setup and Build
+        uses: actions/setup-node@v2
+        with:
+          node-version: '12'
+      - run: |
+          cd calibrate/
+
+          export REACT_APP_BASE_URL=${{ secrets.REACT_APP_BASE_URL_PROD }}
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-frontends-to-production.yml:202]: @NoahNsimbe, kindly update the repo secrets with `REACT_APP_BASE_URL_PROD` accordingly

File: .github/workflows/deploy-frontends-to-production.yml [hunk 2]
```diff
@@ -133,6 +140,9 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
       
+      - name: Set up Docker Buildx
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-frontends-to-production.yml:143]: Meant to speed up the build process

File: .github/workflows/deploy-frontends-to-staging.yml [hunk 3]
```diff
@@ -196,7 +199,7 @@ jobs:
       - run: |
           cd calibrate/
 
-          export REACT_APP_BASE_URL=${{ secrets.REACT_APP_BASE_URL }}
+          export REACT_APP_BASE_URL=${{ secrets.REACT_APP_BASE_URL_STAGE }}
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-frontends-to-staging.yml:202]: @NoahNsimbe, kindly update the repo secrets with `REACT_APP_BASE_URL_STAGE` accordingly

File: k8s/calibrate/prod-airqo-calibrate-app.yaml [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    name: airqo-calibrate-app 
+  minReplicas: 1
+  maxReplicas: 2
+  metrics:
+  - type: Resource
+    resource:
+      name: cpu
+      target:
+        type: Utilization
+        averageUtilization: 70
```
Comment (123MwanjeMike) [inline k8s/calibrate/prod-airqo-calibrate-app.yaml:53]: Spin up new pod when usage of current pod exceeds 70%

File: k8s/calibrate/prod-airqo-calibrate-app.yaml [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+        app: prod-calibrate-app
+    spec:
+      containers:
+        - name: prod-calibrate-app
+          image: us.gcr.io/airqo-250220/airqo-calibrate-app:latest
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 80
+              name: prod-calibrate
+          resources:
```
Comment (123MwanjeMike) [inline k8s/calibrate/prod-airqo-calibrate-app.yaml:26]: Resource capping for each pod

Comment (NoahNsimbe) [reply k8s/calibrate/prod-airqo-calibrate-app.yaml:26]: Great work @123MwanjeMike

File: k8s/calibrate/prod-airqo-calibrate-app.yaml (hunk 4 above)
Comment (NoahNsimbe) [reply k8s/calibrate/prod-airqo-calibrate-app.yaml:53]: Thanks @123MwanjeMike

File: .github/workflows/deploy-frontends-to-staging.yml (hunk 3 above)
Comment (NoahNsimbe) [reply .github/workflows/deploy-frontends-to-staging.yml:202]: @123MwanjeMike I think these were already configured by Daniel.

File: .github/workflows/deploy-frontends-to-production.yml (hunk 1 above)
Comment (NoahNsimbe) [reply .github/workflows/deploy-frontends-to-production.yml:202]: https://github.com/airqo-platform/AirQo-frontend/pull/574#discussion_r846976467

**Review by NoahNsimbe — APPROVED** [review]
LGTM

**Review by Codebmk — APPROVED** [review]

**Pull request merged**